ext/v8/object-template.h

```diff
@@ -43,6 +55,88 @@ namespace rr {
 
       return Qnil;
     }
+
+    static VALUE SetAccessor(int argc, VALUE argv[], VALUE self) {
```

Comment:
Well, The "Name" one seemed like a better choice because we already have these types of accessor callbacks implemented and it is more generic. The documentation doesn't seem to suggest they have different codepaths and I think the String one may be the older of the two.